### PR TITLE
GitCommitBear.py: Change regex to shortlog_regex

### PR DESCRIPTION
**Pull request by sahilarora535**
Fixes https://github.com/coala/coala-bears/issues/1217

### CODE REVIEW

File: bears/vcs/git/GitCommitBear.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def check_shortlog(self, shortlog,
         :param shortlog_length:          The maximum length of the shortlog.
                                          The newline character at end does not
                                          count to the length.
```
Comment (Adrianzatreanu) [inline bears/vcs/git/GitCommitBear.py]: could you please enclose in double backticks "regex" and "shortlog_regex" in the commit shortlog please? :) 
if character count goes over 50 chars, you can just ommit ".py" from the message

Comment (Adrianzatreanu) [inline bears/vcs/git/GitCommitBear.py]: rest looks good to me

Comment (sahilarora535) [inline bears/vcs/git/GitCommitBear.py]: @Adrianzatreanu Done. Hope it passes this time. :smiley:

**Comment by sims1253** [comment]
ack ee7a46e4b9c44c11c9d832c6f0308af7a5d99686

**Comment by sims1253** [comment]
@rultor merge

**Pull request merged**